stock_picking_group_by_partner_by_carrier_sale_line_position/report/report_deliveryslip.xml

```diff
@@ -27,19 +27,6 @@
 
   </template>
 
-  <template
-        id="report_delivery_document"
-        inherit_id="stock_picking_group_by_partner_by_carrier.report_delivery_document"
-    >
-
-    <!-- Add sale line position on remaining to deliver list -->
-    <xpath expr="//tr[@t-foreach='remainings']/td[1]/span[1]" position="before">
-      <span
-                t-if="remaining.get('sale_order_line')"
-                t-esc="remaining['sale_order_line'].position_formatted"
-            />
-    </xpath>
-
-  </template>
+  <!-- Deliver list does not have remainings any more. -->
```

Comment:
This shouldn't be removed, some improvements done on `13.0` haven't been ported to `14.0`, I started to port all missing fix/features on the base module there:
- https://github.com/OCA/stock-logistics-workflow/pull/908
I block the current one until the PR linked is fixed/merged.